samples/msal-node-samples/ElectronSystemBrowserTestApp/README.md

```diff
@@ -99,35 +109,44 @@ npm start
-
-```javascript
- "use static";
+To demonstrate best security practices, this Electron sample uses a custom URL scheme `msal{Your_Application/Client_Id}://auth` that will launch the app when the URL with that scheme is visited.
 
-import auth from "../config/auth.json" assert { type: "json" };
+Using the `acquireTokenInteractive` API, the desktop application starts the [Authorization code flow with PKCE flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow) by launching the system browser using a loopback server to redirect to `successTemplate.html` with the authorization code in the browser. The browser will launch the application upon receiving the authorization code from the **AzureAD**.
 
-window.addEventListener("DOMContentLoaded", () =>  {
-    let redirectUri = auth.customURLScheme;
-
-    if (window.location.search) {
-        redirectUri += window.location.search;
-        window.history.pushState({}, document.title, "/");
+```typescript
+ async getTokenInteractive(
+        tokenRequest: SilentFlowRequest
+    ): Promise<AuthenticationResult> {
+        try {
+            const openBrowser = async (url: any) => {
+                await shell.openExternal(url);
+            };
+
+            const interactiveRequest: InteractiveRequest = {
+                ...tokenRequest,
+                openBrowser,
+                successTemplate: (
+                    await promises.readFile("./public/successTemplate.html")
+                ).toString(),
+                errorTemplate: (
+                    await promises.readFile("./public/errorTemplate.html")
+                ).toString(),
+            };
+
+            const authResponse =
+                await this.clientApplication.acquireTokenInteractive(
+                    interactiveRequest
+                );
+            return authResponse;
+        } catch (error) {
+            throw error;
+        }
     }
+```
 
-    document.getElementById("SignIn").onclick = () => {
-        window.location.href = redirectUri;
-    };
-});
+#### Redirect page
 
-```
+We used a loopback server to host the static web pages [successTemplate](./public/successTemplate.html) and [errorTemplate](./public/errorTemplate.html.html) to serve as the redirect pages. The `successTemplate.html` will receive the authorization code as a query parameter from **AzureAD** and on button click will trigger the Custom URL Scheme `msal{Your_Application/Client_Id}://auth` to launch the application. Deploying the static web page and using it as a redirect page for the electron app is recommended. For more information on hosting a static website on **Azure Storage**, please see [Host a static website on Blob Storage](https://docs.microsoft.com/azure/storage/blobs/storage-blob-static-website-host)
```

Comment:
I don't think this is true or relevant anymore

Reply:
Removed